README.md

```diff
@@ -13,20 +13,12 @@ All projects hosted on this domain, will be found in the /apps folder.
 
 ## Requirements
 
-1. This project requires [pnpm](https://pnpm.io/installation) (yet another npm/yarn alternative)
+This project requires [pnpm](https://pnpm.io/installation) (yet another npm/yarn alternative)
 
 ```
 curl -fsSL https://get.pnpm.io/install.sh | sh -
```

Comment:
I would just delete the info on how to install `pnpm`. People who cannot tell that this repo uses `pnpm` from the lockfiles should probably not touch the code in the first place...